CotEditor/Sources/ScriptManager.swift

```diff
@@ -254,6 +321,20 @@ final class ScriptManager: NSObject, NSFilePresenter {
     }
     
     
+    private func loadScriptInfo(at url: URL) {
```

Comment:
The `bug` section on documentation is added.
IMO that condition will never be happen without macOS's bug while sdef is sane.